Common/UnitDefinitions/Torque.json

```diff
@@ -91,6 +91,42 @@
         }
       ]
     },
+    {
+      "SingularName": "GramForceMillimeter",
+      "PluralName": "GramForceMillimeters",
+      "FromUnitToBaseFunc": "x*0.0000098066502",
```

Comment:
These functions are preferably computations instead of constants, to obtain the highest possible precision.
I assume the gravitational constant multiplied by some power of 10 should be used here?

Reply:
This time I traveled to  
[unitconverters.net ](url)
and did the conversion🤣